packages/sanity-cms-loader/src/index.ts

```diff
@@ -0,0 +1,284 @@
+import DataLoader, { Options } from 'dataloader';
+import { createClient, SanityClient } from '@sanity/client';
+import { map, partition } from 'lodash';
+import { promises as fs } from 'fs';
+import { join, dirname } from 'path';
+import * as ts from 'typescript';
+import { getWinstonLogger } from '@last-rev/logging';
+import Timer from '@last-rev/timer';
+import { ItemKey, ContentfulLoaders, FVLKey, RefByKey } from '@last-rev/types';
+import LastRevAppConfig from '@last-rev/app-config';
+
+const convertSanityDoc = (doc: any) => {
+  if (!doc) return null;
+  const { _id, _type, _updatedAt, ...fields } = doc;
+  return {
+    sys: { id: _id, updatedAt: _updatedAt, contentType: { sys: { id: _type } } },
+    fields
+  };
```

Comment:
I asked for this to prevent changing our extensions. It could be updated to map to a normalized CMS agnostic model instead.